kubernetes-ingress.md

```diff
@@ -0,0 +1,150 @@
+# Ingress
+[⬅️ Back to Kubernetes overview](README.md)
+
+
+## Setup ingress controller
+![Ingress Controller](./kubernetes/ingress.png)
+
+### When using kind:
+From Kind [Documentation](https://kind.sigs.k8s.io/docs/user/ingress/)
+1. Setup Cluster
+
+```sh
+kind delete cluster #if one is alrady created
```

Comment:
We should explain why we need this: *Unfortunately, the only point in time during which we can set static port mappings to a kind cluster is during its creation. Because of that, we need to recreate the cluster here, and add the required attributes.*